monkey365.psm1

```diff
@@ -23,7 +23,7 @@ $internal_modules = @(
     'core/modules/monkeyjob',
     'core/modules/monkeyruleset',
     'core/modules/psocsf',
-    'core/modules/monkeyOutput'
+    'core/modules/monkeyoutput'
```

Comment:
Ok, so you are renaming lots of files (monkeyOutput --> monkeyoutput directory) and then renaming the internal_module path to match? If we want to keep this PR to an state where we can submit the changes back to monkey365 we can not do pointless changes like this.

Can you revert that directory renaming please?

Reply:
Then we cannot run on Ubuntu

Reply:
really? Make a PR for that fix only - we will submit it to monkey365 for inclusing on the main repo

Reply:
so two PRs one for CIS4.0 for ourselve and another (different branch) for this

Reply:
But this might not working properly on Windows. I havent tested on pwsh windows.